### PR TITLE
added feature list to README

### PR DESCRIPTION
**Pull request by FxKu**
to give a quick overview what the operator is capable of

### CODE REVIEW

**Comment by RafiaSabih** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -13,6 +13,39 @@ clusters on Kubernetes (K8s) powered by [Patroni](https://github.com/zalando/spi
 It is configured only through manifests to ease integration into automated CI/CD
 pipelines with no access to Kubernetes directly.
 
+### Operator features
+
+* Rolling updates on cluster changes
```
Comment (Jan-M) [inline README.md]: Postgres cluster changes

File: README.md [hunk 2]
```diff
@@ -13,6 +13,39 @@ clusters on Kubernetes (K8s) powered by [Patroni](https://github.com/zalando/spi
...
+* Volume resize without Pod restarts
+* Cloning Postgres clusters
+* Logical Backups to S3 Bucket
+* Standby cluster from S3 WAL archive
+* Configurable for non-cloud environments
+* UI to create and edit cluster manifests
```
Comment (Jan-M) [inline README.md]: Postgres cluster manifests

File: README.md [hunk 3]
```diff
@@ -13,6 +13,39 @@ clusters on Kubernetes (K8s) powered by [Patroni](https://github.com/zalando/spi
...
+
+### PostgreSQL features
+
+* Supports PostgreSQL 9.3 to 12
```
Comment (Jan-M) [inline README.md]: Spilo does not contain lower than 9.6 I think.

We should also not advertise, we can make it 9.6+ or 10+ to not having to deal with older versions.

**Comment by Jan-M** [comment]
Tiny remarks, otherwise a good step forward for the first page.

**Comment by Jan-M** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Pull request merged**